### ISSUE TITLE
Couple of questions regarding per channel multipliers and bottleneck blocks

### ISSUE THREAD

**Issue by hanspinckaers** [hongyi-zhang/Fixup#3]
Hi there,

The code seems to train well, thanks for the implementation! 

I wonder though if you tried adding multiplier / bias per channel  (like batchnorm) instead of single scalars?
Also I see you use the BasicBlock of Resnet, also for the bigger Resnets. Did you try the Bottleneck block?

**Comment by hongyi-zhang** [comment]
Hi Hans,

Thanks for asking!

- We did try adding multiplier / bias per channel, but found that it led to worse training error (which may suggest the loss landscape was not as good).

- We have the Bottleneck block version available in the ImageNet ResNet code for fixup: https://github.com/hongyi-zhang/Fixup/blob/4351e4d939654d9b8a5ceab20b43005055b0c0e2/imagenet/models/fixup_resnet_imagenet.py#L55-L93

**Comment by hongyi-zhang** [comment]
Closing the issue for now. Let me know if you have any questions!
